packages/common/src/core/schemas/internal/enrichEntity.ts

```diff
@@ -0,0 +1,87 @@
+import { IGroup } from "@esri/arcgis-rest-types";
```

Comment:
wondering where this should live - should it live here within the `schemas` directory and only be leveraged in the context of `IWithEditorBehavior`, or should it live up a level and be a util that can be called from anywhere to enrich an entity with various information?